packer/configure-windows-vhd.ps1

```diff
@@ -77,47 +113,63 @@ function Install-WindowsPatches
 
     $patchUrls = @()
 
-        foreach ($patchUrl in $patchUrls)
-        {
-            $pathOnly = $patchUrl.Split("?")[0]
-            $fileName = Split-Path $pathOnly -Leaf
-            $fileExtension = [IO.Path]::GetExtension($fileName)
-            $fullPath = [IO.Path]::Combine($env:TEMP, $fileName)
+    foreach ($patchUrl in $patchUrls)
+    {
+        $pathOnly = $patchUrl.Split("?")[0]
+        $fileName = Split-Path $pathOnly -Leaf
+        $fileExtension = [IO.Path]::GetExtension($fileName)
+        $fullPath = [IO.Path]::Combine($env:TEMP, $fileName)
 
-            switch ($fileExtension)
+        switch ($fileExtension)
+        {
+            ".msu"
             {
-                ".msu"
+                Write-Log "Downloading windows patch from $pathOnly to $fullPath"
+                Invoke-WebRequest -UseBasicParsing $patchUrl -OutFile $fullPath

```

Comment:
Not needed now, but curl.exe is faster than invoke-webrequest

Reply:
actually - right now this will also have the progress bar enabled which is a 10x slowdown. Can you switch to curl or change the behavior like this? https://github.com/PowerShell/PowerShell/issues/2138#issuecomment-251165868

Reply:
or grab this code https://github.com/Azure/aks-engine/blob/26383d7e5a9f6868713d2d887ec5956cfa5bdd2a/parts/k8s/kuberneteswindowsfunctions.ps1#L12

Reply:
I'm setting the progress preference to 'SilentlyContinue' at the beginning of the script.